### DISCUSSION TITLE
updating the topology information in kvrocks does not change the replication relationships

### DISCUSSION THREAD

**Discussion by quan-kuang** [apache/kvrocks#2616] [Q&A]
I have already removed the slave node using the setnodes command, but I can still see the instance information through the info
![企业微信截图_17295785146549](https://github.com/user-attachments/assets/6d63759e-06e8-410a-8783-3066bc9b6067)
How should I remove this master-slave relationship?

**Comment by git-hulk** [comment]
You should update the cluster topology to the slave node: `10.177.10.156` as well, and it will disconnect with its master node.

**Reply by quan-kuang** [reply]
I have already updated the topology on the slave node 10.177.10.156, but the master node still shows the association
![企业微信截图_17295804236857](https://github.com/user-attachments/assets/04f22e50-0c18-420c-ada0-11142bd3ac79)
I tried deleting the slave node, and the master-slave relationship returned to normal. Can't the relationship be removed just by modifying the topology?

**Reply by git-hulk** [reply]
I see, it should be a bug in the cluster slave node. It didn't remove the master after it's removed from the cluster topology.

**Reply by weim0000** [reply]
if you want to remove the slave node 10.177.10.156 from the topology, you can exec command "cluster reset" on 10.177.10.156, then 10.177.10.156 will disconnect for it's master, and clear the topology on myself

**Reply by git-hulk** [reply]
Yes, that's a good way to remove the offline node. For Kvrocks, it should automatically do this as well.
